src/components/FormSquadScreen.js

```diff
@@ -86,15 +121,29 @@ const FormSquadScreen = ({ allUsers, userGames }) => {
   return (
     <View style={styles.container}>
       <SelectDropdown
-        data={currentUserGames.map((game) => game.game_title)}
+        data={userGames.map((game) => game.game_title)}
         defaultButtonText={"Select Game"}
         onSelect={(selectedGame) => handleSelectGame(selectedGame)}
-        searchInputTxtColor={{}}
         buttonStyle={styles.selectGameBtnStyle}
         buttonTextStyle={styles.selectGameBtnTextStyle}
         dropdownStyle={styles.selectGameDropdownStyle}
         rowTextStyle={styles.selectGameRowTextStyle}
       />
+      <Pressable
+        onPress={() => setCompetitive(!competitive)}
+        style={styles.competitiveBtn}
+      >
+        <Text style={styles.competitiveTxt}>
+          {competitive ? "Competitive" : "Casual"}
+        </Text>
+        <View
+          style={
+            competitive
+              ? styles.competitiveBoxCompetitive
+              : styles.competitiveBoxCasual
+          }
+        ></View>
+      </Pressable>
```

Comment:
This is the button that creates the competitive / casual option

Reply:
totally forgot about this option, thank you for adding it in!